package/ntrip_daemon/src/ntrip_settings.c

```diff
@@ -150,24 +151,57 @@ void ntrip_stop_processes()
   }
 }
 
+/* Move to libpiksi as generic function */
+static bool file_read_value(char *file_path) {
+  bool val = false;
+
+  if (access(file_path, F_OK) == -1) {
+    /* File is not created yet, system is most likely still booting */
+    piksi_log(LOG_DEBUG, "%s doesn't exist", file_path);
+    return val;
+  }
+
+  FILE* fp = fopen(file_path, "r");
+  if (fp == NULL) {
+    piksi_log(LOG_WARNING|LOG_SBP,
+              "Failed to open %s: errno = %d",
+              file_path,
+              errno);
+    return val;
+  }
+
+  val = ('1' == fgetc(fp));
+  fclose(fp);
+
+  return val;
+}
+
 static int ntrip_notify(void *context)
 {
   (void)context;
 
+  /* If NTRIP was enabled before notify call and ntrip_enabled is still true,
+   * it means that user is trying to modify some other NTRIP related setting
+   * than ntrip_enabled -> reject and instruct. */
+  if (file_read_value(NTRIP_ENABLED_FILE_PATH) && ntrip_enabled) {
+    sbp_log(LOG_WARNING, "NTRIP must be disabled to modify settings");
+    return 1;
+  }
+
   snprintf(ntrip_interval_s, sizeof(ntrip_interval_s), "%d", ntrip_interval);
   ntrip_rev1gga_s[0] = ntrip_rev1gga ? 'y' : 'n';
 
   for (size_t i = 0; i < ntrip_processes_count; i++) {
 
     ntrip_stop_process(i);
 
-    // TODO: Remove the need for these by reading from control socket
+    /* TODO: Remove the need for these by reading from control socket */
```

Comment:
@silverjam is this TODO still relevant? maybe @pmiettinen can implement while he's touching this code

Reply:
TODO is still relevant, but probably not germane for this PR, we could add a ticket for it though, basically want to re-use to the command socket that's used for `ntrip_daemon --reconnect` to make an `ntrip_daemon --is-enabled` so we don't have maintain these files.